package.json

```diff
@@ -27,7 +27,8 @@
     "preversion": "npm run lint",
     "release:patch": "npm version patch && npm publish && git push --follow-tags",
     "release:minor": "npm version minor && npm publish && git push --follow-tags",
-    "release:major": "npm version major && npm publish && git push --follow-tags"
+    "release:major": "npm version major && npm publish && git push --follow-tags",
+    "test": "jest"
```

Comment:
actually i think there is no need for jest right now, `scripts/update-size.js` should require `.size-limit.js` and run `sizeLimit` with required config (this is no functionality change for now, just unifying the size-limit config)

`.size-limit.js` will be required automatically when running `size-limit` CLI tool and this is the one that should be used for now on travis, this is described here https://github.com/ai/size-limit#usage

Reply:
Ok @Andarist  let me update my commit!